base/reducedim.jl

```diff
@@ -43,33 +44,24 @@ function reduced_indices0(inds::Indices{N}, d::Int) where N
     end
 end
 
-function reduced_indices(inds::Indices{N}, region) where N
```

Comment:
I initially started modifying this code to fix a problem but then transitioned to the changes above. Still, this seemed like old code in need of some cleanup.